math/mathcore/inc/TStatistic.h

```diff
@@ -43,10 +43,13 @@ class TStatistic : public TObject {
    Double_t    fM2;      ///< Second order momentum
    Double_t    fMin;     ///< Minimum value in the Tstatistic object
    Double_t    fMax;     ///< Maximum value in the TStatistic object
+   Double_t    fSum;     ///< Sum of the values in the TStatistic object
 
 public:
 
-   TStatistic(const char *name = "") : fName(name), fN(0), fW(0.), fW2(0.), fM(0.), fM2(0.), fMin(TMath::Limits<Double_t>::Max()), fMax(TMath::Limits<Double_t>::Min()) { }
+   TStatistic(const char *name = "")
+      : fName(name), fN(0), fW(0.), fW2(0.), fM(0.), fM2(0.), fSum(0.)
```

Comment:
Can we (in a subsequent PR) not make those in-class initializers?
```
Double_t    fSum{0.};     ///< Sum of the values in the TStatistic object
```
etc. With that we'd be left with `TStatistic() = default` and `TStatistic(const char *name): TStatistic() {...}`.